door-open-service/node-backend/src/slackInterface.js

```diff
@@ -23,6 +23,19 @@ export default function (config, slackBot, sockServer) {
           "Consider dispatching a drone to pick up a human.");
 
     } else {
+
+      config.doorTimes = { "garage": 25 * 1000 };
```

Comment:
but where, 
config.slackBot ?
config.socketServer?

I believe the best is to add a new "lock" name, and then define 

`export default function (config, slackBot, sockServer,lock)`